pkg/login/social/social.go

```diff
@@ -96,12 +102,20 @@ func NewOAuthService() {
 			TlsClientKey:       sec.Key("tls_client_key").String(),
 			TlsClientCa:        sec.Key("tls_client_ca").String(),
 			TlsSkipVerify:      sec.Key("tls_skip_verify_insecure").MustBool(),
+			GroupMappingsFile:  sec.Key("group_mappings_file").String(),
```

Comment:
Don't need to add it to this struct since you only use it below. Init `GroupMappings` per default instead.

```suggestion
			GroupMappings: []setting.OAuthGroupMapping{},
```

Reply:
This change was committed